core/git_command.py

```diff
@@ -159,7 +159,9 @@ def _repo_path(self, throw_on_stderr=True):
         repo_path = view.settings().get("git_savvy.repo_path")
 
         if not repo_path:
-            file_path = self.file_path
+            # if we are in an empty tab for a project self.file_path may
+            # return None in which case we check the windows folder
+            file_path = self.file_path or self.window.folders()[0]
```

Comment:
If you look a few lines down, the window's folders are already checked.  The issue in #128 is that the method is that `None` is passed to a method that expected a string.  I've just pushed a fix for #128 and #129.